### PR TITLE
[NCL-7777] Fix Content jumping issue

### PR DESCRIPTION
**Pull request by DnsZhou**

### CODE REVIEW

**Review by matedo1 — COMMENTED** [review]
See jira comments

**Comment by DnsZhou** [comment]
H @matedo1 , changes applied and see jira for latest gifs.

**Comment by matedo1** [comment]
Hi @patrikk0123 , could you please finish code review for this one? 

@DnsZhou could you please squas your commits into one or two? Nine commits seem to be a little bit confusing in this case as finally just a few simply lines were modified.

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { ReactElement } from 'react';
 import styles from './Tabs.module.css';
 
 const stickyOnBreakpoint: PageSectionProps['stickyOnBreakpoint'] = { default: 'top' };
+const tabsMinHeight: React.CSSProperties = { minHeight: '45px' };
```
Comment (patrikk0123) [inline src/components/Tabs/Tabs.tsx:9]: Move it into `Tabs.module.css` file (class tabs).

File: src/components/ProductMilestonePages/ProductMilestonePages.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ import * as productMilestoneApi from 'services/productMilestoneApi';
 
 import { generatePageTitle } from 'utils/titleHelper';
 
+const labelMinWidth: React.CSSProperties = { minWidth: '20px', minHeight: '15px' };
```
Comment (patrikk0123) [inline src/components/ProductMilestonePages/ProductMilestonePages.tsx:19]: `labelStyles`

**Review by patrikk0123 — COMMENTED** [review]
Some code comments.

**Comment by DnsZhou** [comment]
I have force-pushed the changes into one commit and extracted the component. See jira NCL-7777 for gifs that fixed the error.

File: src/components/ProductMilestonePages/ProductMilestonePages.tsx [hunk 3]
```diff
@@ -52,13 +52,9 @@ export const ProductMilestonePages = ({ children }: PropsWithChildren<IProductMi
...
       <TabsItem url="delivered-artifacts">
         Delivered Artifacts{' '}
-        <Tooltip content="Total Count">
-          <Label>
-            <ServiceContainerLoading {...serviceContainerArtifacts} variant="inline" title="Delivered Artifacts Count">
-              {serviceContainerArtifacts.data?.totalHits}
-            </ServiceContainerLoading>
-          </Label>
-        </Tooltip>
+        <TabsLabel serviceContainer={serviceContainerArtifacts} tooltip={'Total Count'} title={'Delivered Artifacts Count'}>
```
Comment (matedo1) [inline src/components/ProductMilestonePages/ProductMilestonePages.tsx:55]: Using 
```tsx
<TabsLabel serviceContainer={serviceContainerArtifacts} title="Delivered Artifacts Count">
``` 
etc. should be enough?

File: src/components/Tabs/Tabs.module.css [hunk 4]
```diff
@@ -2,3 +2,21 @@
   top: 0;
   z-index: 100;
 }
+
+.page-section-tabs :global(.pf-c-tabs) {
+  min-height: 42px;
+}
+
+.page-section-tabs :global(.pf-c-label) {
```
Comment (matedo1) [inline src/components/Tabs/Tabs.module.css:10]: I can see you decided to implement `TabsLabel` component I proposed, in this case you should move also part of the related stylesheets (nested ones) into new file `TabsLabel.module.css`:

```css
.tabs-label :global(.pf-c-label) { /* ... */ }

/* etc */
```

File: src/components/Tabs/TabsLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import { PropsWithChildren } from 'react';
+
+import { IServiceContainer } from 'hooks/useServiceContainer';
+
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+
+interface ITabsLabelProps {
+  serviceContainer: IServiceContainer;
+  title: string;
+  tooltip: string;
```
Comment (matedo1) [inline src/components/Tabs/TabsLabel.tsx:11]: remove, `title` is enough

**Review by matedo1 — CHANGES_REQUESTED** [review]

**Review by matedo1 — CHANGES_REQUESTED** [review]

File: src/components/Tabs/TabsLabel.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+interface ITabsLabelProps {
+  serviceContainer: IServiceContainer;
+  title: string;
+}
+
+export const TabsLabel = ({ serviceContainer, title, children }: PropsWithChildren<ITabsLabelProps>) => {
+  return (
+    <span className={styles['page-section-tabs-label']}>
+      <Label>
```
[the remark this answers is not in the document]
Comment (DnsZhou) [reply src/components/Tabs/TabsLabel.tsx]: added back

**Review by patrikk0123 — APPROVED** [review]
I approve, just please squash commits.

**Comment by DnsZhou** [comment]
Hi @patrikk0123 , there are only 3 commits now, it was already squashed in the last force push. wold that be okay?

**Comment by patrikk0123** [comment]
> Hi @patrikk0123 , there are only 3 commits now, it was already squashed in the last force push. wold that be okay?

Count does not matter that much, but commits contradict themselves - add tooltip; bring tooltip back.
The final branch before merging should really be divided based on content and historical changes as the branch was developed need to be hidden by squashing.

**Comment by DnsZhou** [comment]
squashed and merged. Thanks @patrikk0123

**Pull request merged**